jekyll/asset/script/main.js

```diff
@@ -419,22 +444,40 @@
       postingData.requiredSkills = requiredSkills;
       postingData.preferredSkills = preferredSkills;
 
-      // Also include Certs
+      var certificationsNeeded = [];
+      $('input[name=certNeeded]').each(function() {
+        if(this.value) {
+          certificationsNeeded.push({name: this.value});
+        }
+      });
+      postingData.certificationsNeeded = certificationsNeeded;
 
       var trainingOfferedEl = $("input:radio[name=training]:checked");
       if(trainingOfferedEl) postingData.trainingOffered = trainingOfferedEl.val();
 
       return templates.fullJobPosting.render(postingData, templates);
     }
 
+    function collectAddedSkills() {
+      $('form + input').each(function(){
+        if (this.value) {
+          var id = this.name.substring(7, this.name.length);
+          currentSkillSet[id] = this.value;
+        }
+      });
+    }
+
     function renderSkillSet(name, skills, id) {
       var skillSet = {
         id: id,
         type: name,
         skills: skills
       };
 
-      $("#" + id)[0].innerHTML = templates.skillSet.render(skillSet, templates);
+      var element = $("#" + id)[0];
+      element.innerHTML = templates.skillSet.render(skillSet, templates);
+      cuff.controls.addSkillsButton = duplicateSkillControl;
```

Comment:
Actually, unignore it. Let's try moving it to initControls but leave the cuff(element). I think it'll work because when you run cuff it'll bind correctly. The only risk is that b/c there are no data-control attributes with that function defined yet, it might complain.